app/src/main/res/layout/activity_dog_owner_profile.xml

```diff
@@ -25,12 +25,175 @@
         app:civ_border="true"
         app:civ_border_color="@color/colorPrimaryDark" />
 
-    <ExpandableListView
-        android:id="@+id/lvExp"
-        android:layout_height="match_parent"
+    <LinearLayout
+        android:id="@+id/stats_profile"
         android:layout_width="match_parent"
-        android:layout_marginTop="14dp"
+        android:layout_height="wrap_content"
         android:layout_below="@id/profile_picture"
-        android:groupIndicator="@null"/>
+        android:orientation="vertical"
+        android:layout_marginTop="20dp">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/stats"
+            android:textSize="24sp"
+            android:textAlignment="center"/>
+
+        <LinearLayout
```

Comment:
Would've loved to just use a grid layout here. But apparently grid layout doesn't support evenly distributing column width.